source/includes/fact-viewing-favorite-queries.rst

```diff
@@ -12,4 +12,21 @@
 
    .. step:: Click the :guilabel:`Favorites` button in the past queries pane.
 
-      Select favorite queries.
+      Select your favorited queries.
+
+      - If your favorited query is a query, only a :guilabel:`Filter` 
```

Comment:
[question] Is there another term we can use in place of "favorited query"? I feel like it's slightly odd that all entries under favorite queries are not technically "quereis"